slack-api-model/src/main/java/com/slack/api/model/AppStep.java

```diff
@@ -0,0 +1,11 @@
+package com.slack.api.model;
```

Comment:
Can you move this to `com.slack.api.model.workflow` package?

Reply:
Sure, definitely.